content/api.md

```diff
@@ -4,15 +4,150 @@ title: API usage
 ---
 ## Introduction
 
-The data collected for the [2FA Directory](https://2fa.directory) website is also available as JSON files to enable developers to use it in their programs. The API with the highest version number is recommended since older versions might not include all available information.
+The data collected for the [2FA Directory][site_url] website is also available as JSON files for developers to use in their programs.
 
-### Caching
+## Supported versions
 
-If you intend to query our JSON files often and with a lot of traffic, you may be blocked by Cloudflare, our reverse proxy provider. We therefore recommend that you cache the files locally for any significant traffic cases.
+The table below provides the End of Life (EOL) dates for different 2FA Directory API versions. Please refer to this table to ensure you are using a supported version.
 
-### Avoid downloading unnecessary data
+|     Version      | Active Support | End of Life Date |
+|:----------------:|:--------------:|:----------------:|
+| [Version 4](#v4) |       ✅        |       N/A        |
+| [Version 3](#v3) |       ✅        |       N/A        |
+| [Version 2](#v2) |       ❌        |    2023-08-01    |
+| [Version 1](#v1) |       ❌        |    2023-07-01    |
 
-If you only intend to use a specific dataset, like all sites supporting RFC-6238, we recommend using the URI which lists _just_ that. See [URIs](#uris) for available paths. The smaller, the better.
+## Things to note
+
+When using the 2FA Directory data, there are a few important considerations to keep in mind to ensure compliance and optimal performance.
+
+#### Attribution
+
+The data for the 2FA Directory is licensed under an [MIT license][license] which requires attribution when using the data.
+If the data collected from the 2FA Directory is visible to your users, please provide attribution. This helps acknowledge the source and supports the ongoing maintenance of the directory.  
+At a minimum, please include the following attribution:
+
+> Data sourced from [2FA Directory][site_url] by [2factorauth][repo_url].
+
+#### Caching
+
+It is advisable to cache the files locally to avoid potential service interruptions, especially if you intend to query our JSON files frequently or generate significant traffic. Cloudflare, our reverse proxy provider, may block excessive traffic, so local caching ensures your access remains uninterrupted and reduces server load.
+
+#### Avoid downloading unnecessary data
+
+It is recommended that you download only the specific datasets you need to maximize efficiency and minimize data transfer. For example, if you only require information about sites supporting RFC-6238, use the URI that lists only those sites. This approach conserves bandwidth and accelerates data processing.
```

Comment:
I know the current page only has `RFC-6238`, but should we say something like `TOTP (RFC-6238)`? Also, is it `RFC-6238` or `RFC 6238`?